Monika After Story/game/script-holidays.rpy

```diff
@@ -4845,3 +4752,1114 @@ label greeting_gone_over_f14_normal_plus:
     m 5ekbsa "Thank you for making sure we had a wonderful Valentine's Day, [player]~"
     $ persistent._mas_f14_gone_over_f14 = False
     return
+
+############################### 922 ###########################################
+# [HOL060]
+#START:
+
+#Moni's bday
+define mas_monika_birthday = datetime.date(datetime.date.today().year, 9, 22)
+
+#922 mode
+default persistent._mas_bday_in_bday_mode = False
+
+#Date related vars
+default persistent._mas_bday_on_date = False
+default persistent._mas_bday_date_count = 0
+default persistent._mas_bday_date_affection_gained = 0
+default persistent._mas_bday_gone_over_bday = False
+
+#Suprise party bits and bobs
+default persistent._mas_bday_sbp_reacted = False
+default persistent._mas_bday_confirmed_party = False
+
+#Bday visuals
+default persistent._mas_bday_visuals = False
+
+#Need to store the name of the file chibi writes
+default persistent._mas_bday_hint_filename = None
+
+#Time spent tracking
+default persistent._mas_bday_opened_game = False
+default persistent._mas_bday_no_time_spent = True
+default persistent._mas_bday_no_recognize = True
+default persistent._mas_bday_said_happybday = False
+
+############### [HOL060]: HISTORY
+init -810 python:
+    store.mas_history.addMHS(MASHistorySaver(
+        "922",
+        datetime.datetime(2020, 1, 6),
+        {
+            "_mas_bday_in_bday_mode": "922.bday_mode",
+
+            "_mas_bday_on_date": "922.on_date",
+            "_mas_bday_date_count": "922.actions.date.count",
+            "_mas_bday_date_affection_gained": "922.actions.date.aff_gained",
+            "_mas_bday_gone_over_bday": "922.gone_over_bday",
+
+            "_mas_bday_sbp_reacted": "922.actions.surprise.reacted",
+            "_mas_bday_confirmed_party": "922.actions.confirmed_party",
+
+            "_mas_bday_opened_game": "922.actions.opened_game",
+            "_mas_bday_no_time_spent": "922.actions.no_time_spent",
+            "_mas_bday_no_recognize": "922.actions.no_recognize",
+            "_mas_bday_said_happybday": "922.actions.said_happybday"
+        },
+        exit_pp=store.mas_history._bday_exit_pp
```

Comment:
can you add appropriate `start_dt` and `end_dt` for this? This is used in trigger resetting algorithms.